e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts

```diff
@@ -10,31 +11,29 @@ import { Common, setupBrowser } from "../../../utils/common";
 import { UIhelper } from "../../../utils/ui-helper";
 import fs from "fs/promises";
 import { RbacPo } from "../../../support/pageObjects/rbac-po";
-
-// TODO: reenable tests, replace skip with serial
-test.describe.skip(
-  "Test RBAC plugin: load permission policies and conditions from files",
-  () => {
-    let common: Common;
-    let uiHelper: UIhelper;
-    let page: Page;
-
-    test.beforeAll(async ({ browser }, testInfo) => {
-      page = (await setupBrowser(browser, testInfo)).page;
-
-      uiHelper = new UIhelper(page);
-      common = new Common(page);
-      await common.loginAsGithubUser();
-      await uiHelper.openSidebarButton("Administration");
-      await uiHelper.openSidebar("RBAC");
-      await uiHelper.verifyHeading("RBAC");
+import { RhdhAuthApiHack } from "../../../support/api/rhdh-auth-api-hack";
+import RhdhRbacApi from "../../../support/api/rbac-api";
+import { RbacConstants } from "../../../data/rbac-constants";
+import { PolicyComplete } from "../../../support/api/rbac-api-structures";
+
+/*
+    Note that:
+    The policies generated from a policy.csv or ConfigMap file cannot be edited or deleted using the Developer Hub Web UI. 
+    https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/authorization/managing-authorizations-by-using-the-web-ui#proc-rbac-ui-edit-role_title-authorization
+*/
+test.describe.serial("Test RBAC", () => {
+  test.describe
+    .serial("Test RBAC plugin: load permission policies and conditions from files", () => {
+    test.beforeEach(async ({ page }) => {
+      await new Common(page).loginAsKeycloakUser();
+      await page.goto("/rbac");
```

Comment:
In test at least once we should cover the rbac navigation from sidebar `Administration>RBAC`

Reply:
totally. Just pushed it, thanks!